samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/author/ItemAddListener.java

```diff
@@ -240,6 +240,15 @@ public void processAction(ActionEvent ae) throws AbortProcessingException {
     
     if(iType.equals(TypeFacade.FILL_IN_BLANK.toString())){
 	
+    	int markerError = isErrorMarkersFIB();
+    	if (markerError > 0) {
+    		err =ContextUtil.getLocalizedString("org.sakaiproject.tool.assessment.bundle.AuthorMessages", "pool_badmarkers_error_" + markerError);
+    		context.addMessage(null, new FacesMessage(err));
+    		item.setOutcome("fillInBlackItem");
+    		item.setPoolOutcome("fillInBlackItem");
```

Comment:
Should this me FillInBlankItem instead?

Reply:
Existing references to fillInBlackItem:

`samigo/samigo-app/src/webapp/WEB-INF/faces-config.xml
590:      <from-outcome>fillInBlackItem</from-outcome>

samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/author/StartCreateItemListener.java
212:                        nextpage = "fillInBlackItem";

samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/author/ItemAddListener.java
246:                    item.setOutcome("fillInBlackItem");
247:                    item.setPoolOutcome("fillInBlackItem");

samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/author/ItemModifyListener.java
332:                        nextpage = "fillInBlackItem";`